openapi/admin/userGroupAdministration.yaml

```diff
@@ -0,0 +1,99 @@
+get:
+  tags:
+  - "UserGroupManagement"
+  operationId: "getUsersInGroup"
+  description: "Gets all users in a user group"
+  parameters:
+  - name: "userGroupId"
+    description: "ID of the user group to get users for"
+    in: "query"
+    required: true
+    schema:
+      type: "integer"
+      format: "int64"
+  responses:
+    "200":
+      description: "User group relations for the specified user group"
+      content:
+        application/json:
+          schema:
+            $ref: "../schemas/admin/UserGroupRelations.yaml"
+    "401":
+      $ref: "../responses/UnauthorizedError.yaml"
+    "403":
+      $ref: "../responses/ForbiddenError.yaml"
+post:
+  tags:
+  - "UserGroupManagement"
+  operationId: "addUserToUserGroup"
+  description: "Add a user to a user group"
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: "../schemas/admin/UserGroupRelationCreateRequest.yaml"
+  responses:
+    "200":
+      description: "User group relation for user was successfully created"
+      content:
+        application/json:
+          schema:
+            $ref: "../schemas/DefaultCreateResponse.yaml"
+    "401":
+      $ref: "../responses/UnauthorizedError.yaml"
+    "403":
+      $ref: "../responses/ForbiddenError.yaml"
+put:
+  tags:
+  - "UserGroupManagement"
+  operationId: "updateUserRelationToUserGroup"
+  description: "Add a user to a user group"
+  requestBody:
```

Comment:
Shouldn't this take a `UserGroupRelationId` as parameter?

Using the DefaultXXXResponse indicates something about ids being created, but it's only halfway used.

I don't mind using GroupId/UserId as composite key, but then it should be used throughout this api class.